nautobot/core/celery/__init__.py

```diff
@@ -114,7 +115,7 @@ def nautobot_kombu_json_loads_hook(data):
         logger.debug("Performing nautobot deserialization for type %s", qual_name)
         cls = import_string(qual_name)  # fully qualified dotted import path
         if cls:
-            return cls.nautobot_deserialize(data)
+            return SimpleLazyObject(lambda: cls.objects.get(id=data["id"]))
```

Comment:
Nothing today since this isn't being used but the hope is that it will let us use database objects and querysets as job arguments and deserialize them without having to access the database. Accessing the database when deserializing the NautobotFakeRequest was causing an issue that was fixed in #3105